boe_min.sql

```diff
@@ -0,0 +1,86 @@
+-- For SQLite DBs
+-- conn.executescript(sql_script_string)
+-- minutes DB schema planning
+-- SQLite is loosely typed, meaning you can put almost any data in any column,
+--  regardless of the defined type. (Except for int PKs. I think.)
+;
+
+CREATE TABLE IF NOT EXISTS minutes(
+    min_id INTEGER CONSTRAINT min_id_pk PRIMARY KEY AUTOINCREMENT,
+    min_filename TEXT NOT NULL,
+    min_date TEXT NOT NULL,
+    min_text, TEXT NOT NULL  
+);
+
+CREATE TABLE IF NOT EXISTS contractors(
+    con_id INTEGER CONSTRAINT con_id_pk PRIMARY KEY AUTOINCREMENT,
+    con_name TEXT NOT NULL,
+    con_acc_id TEXT NOT NULL,
+    con_state TEXT NOT NULL,
+    con_address TEXT NOT NULL,
+    con_notes TEXT,
+    con_type_id INTEGER NOT NULL,
+    con_mwbe INTEGER,  -- 0 for no, 1 for yes; allows further specification into the future (e.g., 0 for no, 1+ for different kinds)
+    CONSTRAINT con_type_id_fk FOREIGN KEY (con_type_id) REFERENCES contractor_types(con_type_id),
+    CONSTRAINT con_acc_id_fk FOREIGN KEY (con_acc_id) REFERENCES accounts (acc_id)
+);
+
+CREATE TABLE IF NOT EXISTS contractor_types(
+    con_type_id INTEGER CONSTRAINT con_type_id_pk PRIMARY KEY AUTOINCREMENT,
+    con_type_name TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS departments(
+    dept_id INTEGER CONSTRAINT dept_id_pk PRIMARY KEY AUTOINCREMENT,
+    dept_name TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS applicants(
```

Comment:
Are these applicants applying for "minor privileges"? Those are the only applicants I've seen my readings of the  minutes. 

If those are the applicants you mean, I'm not sure they're contractors. They may just be property owners. We'll have to talk to a subject matter expert to be sure.